### PR TITLE
[17.0][MIG] pos_operating_unit: Migration to 17.0

### PR DESCRIPTION
**Pull request by traitrantrobz**
Depends on :

- [MIG] operating_unit: Migration to 17.0 #667

### CODE REVIEW

**Comment by dzungtran89** [comment]
@traitrantrobz please double check the api `create` and security.

File: pos_operating_unit/views/pos_views.xml [hunk 1]
```diff
@@ -41,8 +33,8 @@
         <field name="model">pos.order</field>
         <field name="inherit_id" ref="point_of_sale.view_pos_order_tree" />
         <field name="arch" type="xml">
-            <field name="session_id" position="before">
-                <field name="config_id" />
+            <field name="session_id" position="after">
```
Comment (dzungtran89) [inline pos_operating_unit/views/pos_views.xml]: @traitrantrobz Could you please share your ideas for this change?

**Comment by dzungtran89** [comment]
> @traitrantrobz please double check the api `create` and security.

@traitrantrobz I just updated the comment.

**Comment by traitrantrobz** [comment]
> @traitrantrobz please double check the api `create` and security.

@dzungtran89 Could you please elaborate more where create function is incorrect?

**Comment by dzungtran89** [comment]
It should be decorated by `model_create_multi`.

**Comment by traitrantrobz** [comment]
> It should be decorated by `model_create_multi`.

Ảnh chụp Màn hình 2024-06-27 lúc 15 32 00
I got it, thanks sir for review!

**Comment by dzungtran89** [comment]
@traitrantrobz could you please double check the code again?

**Comment by traitrantrobz** [comment]
The module pos_operating_unit has been migrated.

**Pull request closed without merging**